Resources/config/dbal.xml

```diff
@@ -100,5 +100,9 @@
             <tag name="console.command" command="doctrine:query:sql" />
         </service>
 
+        <service id="doctrine.ping_command" class="Doctrine\Bundle\DoctrineBundle\Command\Proxy\PingDoctrineCommand">
+            <tag name="console.command" command="doctrine:ping" />
```

Comment:
I'd be in favour of naming this `doctrine:database:ping` - opinions?

Reply:
Sure, i guess everything else has 3 levels so makes sense